packages/flame/lib/src/experimental/fixed_integer_resolution_viewport.dart

```diff
@@ -1,97 +0,0 @@
-import 'dart:math' as math;
-
-import 'package:flame/extensions.dart';
-import 'package:flame/game.dart';
-
-/// This viewport is a very similar to [FixedResolutionViewport], but allows
-/// for better handling of viewing pixel art. The main point is it ensures
-/// sprites align to the physical pixel grid and only scales at integer
-/// intervals. This prevents artifacts or distortion happening to scaled up
-/// pixel art as discussed in https://github.com/flame-engine/flame/issues/1152.
-/// Remember to set [devicePixelRatio] before using the viewport; a good place
-/// to update it is in handleResize of your game, directly from
-/// WidgetBindings.instance.window.devicePixelRatio.
-class FixedIntegerResolutionViewport extends Viewport {
```

Comment:
why remove?

Reply:
Because it's a viewport for the old camera

Reply:
Created #2810 to track this for the new camera, since it was in the experimental section I don't think it matters much that we remove it momentarily (it was used for a specific rendering bug in Flutter).